handlers/get_tasks_next.go

```diff
@@ -0,0 +1,38 @@
+package handlers
+
+import (
+	"database/sql"
+	"log"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetTasksNext(c *gin.Context) {
+
+	db, err := sql.Open("postgres", "postgres://postgres:mysecretpassword@localhost:5432/postgres?sslmode=disable")
+
+	limit := c.Query("limit")
+	if limit == "" {
+		limit = "10"
+	}
+
+	rows, err := db.Query("SELECT * FROM tasks WHERE completed=false AND due_date<=NOW() ORDER BY due_date ASC LIMIT $1", limit)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
Si hay un error, no deberíamos hacer un `log.Fatal(err)`, porque los fatal son muy peligrosos. Si haces un `Fatal` o `panic`, lo que se hará es romper la aplicación.

Aqui la clave sería que pusieras un log informando del error para que el desarrollador pueda consultarlo y a la vez puedes abortar la ejecución y enviarle un estado que no sea 200(ya que 200 significa que ha ido bien).

Para que lo veas más claro, un ejemplo sería

```suggestion
	if err != nil {
		log.Printf("error querying for next tasks: %v \n",err)
		c.AbortWithStatus(http.StatusInternalServerError)
		return
	}
```

El `http.StatusInternalServerError` es el 500, que indica que hubo un error en el servidor.

Reply:
hecho